### ISSUE TITLE
Canal E

### ISSUE THREAD

**Issue by sguinetti** [iptv-org/iptv#4819]
**Channel Name:** Canal E (also Canal E+)
**Country:** PE
**Language:** Spanish
**Website:** Not yet
**Stream URL (optional):** Unknow
**Notes (optional):** Local channel from Rioja. Although it does not have a website, it does have mobile applications and Facebook. page https://play.google.com/store/apps/details?id=canal.e.altomayo and https://play.google.com/store/apps/details?id=canal.enororiente

**Comment by jkemp814** [comment]
No estoy seguro de si este sitio web ayudaría, podría encontrar Canal E o Canal E +

https://www.tvperu.gob.pe/play

**Comment by sguinetti** [comment]
> No estoy seguro de si este sitio web ayudaría, podría encontrar Canal E o Canal E +
> 
> https://www.tvperu.gob.pe/play https://www.gatotv.com/canales_de_tv

Pues lamento decir que estos canales no se encuentran en esas páginas web. Como sabrás como canales locales, así que es poco probables encontrar en la guía. No se si eso es lo que intentas preguntarme.

**Comment by sguinetti** [comment]
This PR changes the name to TV Peru Internacional and differentiate with the domestic signal.

**Comment by jkemp814** [comment]
Sí, eso es lo que estaba tratando de decir. Quizás conduzcan a algo.

**Comment by sguinetti** [comment]
Por cierto, este es la página de Facebook https://www.facebook.com/PRODUCTORAKYJ/ para que tengas más información sobre el canal.

**Comment by jkemp814** [comment]
Bien gracias. Lo comprobaré.

**Comment by LaneSh4d0w** [comment]
Voy a mirar los aplicaciones del Google Play Store. Con un poco de suerte, puedo ver los vinculos.

**Comment by sguinetti** [comment]
> Voy a mirar los aplicaciones del Google Play Store. Con un poco de suerte, puedo ver los vinculos.

Gracias por el apoyo. Más allá de esas sugerencias, tengo el presentimiento que el futuro más canales recurrirán a aplicaciones móviles que páginas web para ver transmisiones en vivo.

**Comment by LaneSh4d0w** [comment]
El source code del Canal E Nor Oriente es... muy muy raro.

Eso es probablemente un sistem de chat

![image](https://user-images.githubusercontent.com/30985701/135730997-58303de9-6824-46b2-b96d-cf3406d286f6.png)

**Comment by sguinetti** [comment]
Parece que desarrollaron a partir de un servicio para crear aplicaciones móviles. Por eso tiene componentes que podrían usar pero que para esta aplicación resulta innecesaria.

**Comment by LaneSh4d0w** [comment]
HOST = http://srv1.mediastreamperu.com:8081/

**Comment by jkemp814** [comment]
https://srv1.mediastreamperu.com:8081/canalenororiente/index.m3u8: Server returned 404 Not Found
https://srv1.mediastreamperu.com:8081/canalealtomayo/index.m3u8: Server returned 404 Not Found

**Comment by LaneSh4d0w** [comment]
Gotcha. I had made some guessing linked to some other channels available from the same server.

**Comment by LaneSh4d0w** [comment]
Ladies and gentleman, we got him.

https://srv1.mediastreamperu.com:8081/8006/index.m3u8

**Comment by LaneSh4d0w** [comment]
![image](https://user-images.githubusercontent.com/30985701/135732379-fd5141de-e545-4852-9616-1be3041c3e16.png)
![image](https://user-images.githubusercontent.com/30985701/135732386-b1777d55-3c18-45e4-8007-2f66cf26fe19.png)

**Comment by sguinetti** [comment]
> https://srv1.mediastreamperu.com:8081/8006/index.m3u8

(in spanish:) ¡Que preciso! Voy a probar en VLC.

**Comment by jkemp814** [comment]
Does that play for you? I notice in your image to the right it saids Not Found. At the top and bottom of the image.

ffplay and VLC I get: https://srv1.mediastreamperu.com:8081/8006/index.m3u8: Server returned 404 Not Found
 
Are you using proxy or VPN?

**Comment by LaneSh4d0w** [comment]
Nope doesn't play. Maybe it's not 24/7.

Le dim. 3 oct. 2021 à 01:20, James E. Kemp ***@***.***> a
écrit :

**Comment by jkemp814** [comment]
I was thinking that.

**Comment by sguinetti** [comment]
As is the case with several local Peruvian channels, the link is a Not 24/7. So this link is taken as is for now.

**Comment by sguinetti** [comment]
By the way, have you tested if the two applications carry the same stream url.

**Comment by jkemp814** [comment]
I get a differnet result when using http://

james@HP-Z840:~$ ffplay -hide_banner -loglevel 24 http://srv1.mediastreamperu.com:8081/canalenororiente/index.m3u8
http://srv1.mediastreamperu.com:8081/canalenororiente/index.m3u8: End of file

james@HP-Z840:~$ ffplay -hide_banner -loglevel 24 https://srv1.mediastreamperu.com:8081/canalenororiente/index.m3u8
[https @ 0x7fe9580017c0] HTTP error 404 Not Found
https://srv1.mediastreamperu.com:8081/canalenororiente/index.m3u8: Server returned 404 Not Found

**Comment by sguinetti** [comment]
I forgot to tell you that each app has two transmissions of channel E. One for Altomayo and one for Rioja (that's why it is called E+) What do you suggest for now? I can add the two links in the PR to include them later.

**Comment by jkemp814** [comment]
That will only hold up the PR. We have to verify somehow these links work.

**Comment by LaneSh4d0w** [comment]
> 
> 
> I get a differnet result when using http://
> 
> james@HP-Z840:~$ ffplay -hide_banner -loglevel 24 http://srv1.mediastreamperu.com:8081/canalenororiente/index.m3u8 http://srv1.mediastreamperu.com:8081/canalenororiente/index.m3u8: End of file
> 
> james@HP-Z840:~$ ffplay -hide_banner -loglevel 24 https://srv1.mediastreamperu.com:8081/canalenororiente/index.m3u8 [https @ 0x7fe9580017c0] HTTP error 404 Not Found https://srv1.mediastreamperu.com:8081/canalenororiente/index.m3u8: Server returned 404 Not Found

the http link gets me a NET_RESET on my side.

**Comment by LaneSh4d0w** [comment]
Found out the srv1.mediastreamperu.com had 2 different ports.

2021 gives us a MediaCP panel.
8081 gives us a Flussonic panel. Seems they work onto 2 different sorts of panels.

**Comment by jkemp814** [comment]
Interesting. The link has to be hidden in there somewhere.

**Comment by LaneSh4d0w** [comment]
8006 and 8008 answer using "vnd.apple.mpegurl". We might have our culprits.

**Comment by sguinetti** [comment]
https://srv1.mediastreamperu.com:8081/8006/index.m3u8 works for now.

**Comment by LaneSh4d0w** [comment]
Works for me too. Seems that my Frida work was correct then :)

**Comment by LaneSh4d0w** [comment]
> **Channel Name:** Canal E (also Canal E+) **Country:** PE **Language:** Spanish **Website:** Not yet **Stream URL (optional):** Unknow **Notes (optional):** Local channel from Altomayo city. Although it does not have a website, it does have mobile applications and Facebook. page https://play.google.com/store/apps/details?id=canal.e.altomayo and https://play.google.com/store/apps/details?id=canal.enororiente with Live TV embed.

Canal E Nororiente is also in srv1.mediastreamperu.com

**Comment by LaneSh4d0w** [comment]
To finish the grand course : https://srv1.mediastreamperu.com:8081/8008/index.m3u8 is Canal Evelyn Nororiente.

**Comment by LaneSh4d0w** [comment]
https://srv1.mediastreamperu.com:8081/8004/index.m3u8 idk what this one is. Also seems to be Canal Evelyn.
